pom.xml

```diff
@@ -95,7 +95,7 @@
         <version.org.codehaus.woodstox.stax2-api>3.1.4</version.org.codehaus.woodstox.stax2-api>
         <version.org.codehaus.woodstox.woodstox-core-asl>4.4.1</version.org.codehaus.woodstox.woodstox-core-asl>
         <version.org.fusesource.jansi>1.9</version.org.fusesource.jansi>
-        <version.org.jboss.aesh>0.33.16</version.org.jboss.aesh>
+        <version.org.jboss.aesh>0.62</version.org.jboss.aesh>
```

Comment:
This doesn't matter for this PR and my comment is off topic, but this is a weird versioning scheme. This project is already being used in high value products with long, long support cycles, so what's the point of aesh still being on major version 0? Given the changes required to use this new version, it's clear 0.6x is not really part of the same major series as 0.3x.